### ISSUE TITLE
5 high severity vulnerabilities

### ISSUE THREAD

**Issue by DarthRevanXX** [bellingard/sonar-scanner-npm#126]
Why installing "sonarqube-scanner" getting below vulnerabilities(npm audit fix doesn't help):
```
node_modules/normalize-url
  cacheable-request  0.1.0 - 6.0.0
  Depends on vulnerable versions of normalize-url
  node_modules/cacheable-request
    got  8.0.0 - 9.5.0
    Depends on vulnerable versions of cacheable-request
    node_modules/got
      download  >=7.0.0
      Depends on vulnerable versions of got
      node_modules/download
        sonarqube-scanner  >=2.1.0
        Depends on vulnerable versions of download
        node_modules/sonarqube-scanner

5 high severity vulnerabilities
```

**Comment by DarthRevanXX** [comment]
added pull request
https://github.com/bellingard/sonar-scanner-npm/pull/127

**Comment by bellingard** [comment]
I've merged #127, thanks @DarthRevanXX
